apps/asap-server/src/entities/group.ts

```diff
@@ -15,11 +14,9 @@ import { parseDate, createURL } from '../utils/squidex';
 
 export const parseGraphQLGroup = (item: GraphqlGroup): GroupResponse => {
   const createdDate = parseDate(item.created).toISOString();
-  const teams: GroupTeam[] = (item.flatData?.teams || []).map((t) => {
-    const team: TeamResponse = parseGraphQLTeam(t, []);
-    delete team.members;
```

Comment:
@johnytiago @peterstarling older TS did not catch this - deleting `team.members` is not allowed because `members` is a mandatory property on `GroupTeam`. Is this safe to remove or why was this in there?

Reply:
@johnytiago @peterstarling ping. I would assume it's not necessary since we're literally just deleting data in conflict with type declarations? And deployment seems to work

Reply:
Oh, the tests fail. I guess I'll throw in destructuring instead of delete, don't need code knowledge for that :sweat_smile: